### ISSUE TITLE
--skip-live option with export seems to still export the .mov?

### ISSUE THREAD

**Issue by spencerc99** [RhetTbull/osxphotos#537]
The full command I'm running is 
```
osxphotos export --download-missing --album fits\ 🧢 --use-photokit --skip-live --skip-original-if-edited --filename "{created.date}:{original_name}" --update --convert-to-jpeg --post-function scripts/transform_exported_fits_into_db.py::post_function --preview ./fits-export
```

I noticed that some .mov files still got exported in the latest batch
![image](https://user-images.githubusercontent.com/14796580/141245608-836d70ca-8508-474f-a981-c0bd202abbb5.png)

any ideas?

**Comment by RhetTbull** [comment]
Are you sure they're live images and not just movie files? Search for the file base name (eg IMG_4857) to see if you can find the matching still image. I'll take a look at the code to see if anything jumps out.

**Comment by RhetTbull** [comment]
I was able to reproduce this behavior.  Seems to be an issue with the `--use-photokit` option as it does not happen with `--download-missing` when `--use-photokit` is not used.

**Comment by RhetTbull** [comment]
@all-contributors add @spencerc99 for bug

**Comment by RhetTbull** [comment]
I think this is fixed in [v0.43.4](https://github.com/RhetTbull/osxphotos/releases/tag/v0.43.4)  Let me know if you still have issues.

**Comment by spencerc99** [comment]
awesome will give it a try thanks!!

--
Spencer Chang
sometimes writing, sometimes reading, often eating.
spencerchang.me ( http://spencerchang.me/ )
